tests/rptest/tests/raft_availability_test.py

```diff
@@ -222,6 +222,22 @@ def test_one_node_down(self):
         the cluster remains available afterwards, and that the expected
         peer takes over as the new leader.
         """
+        # Retry a couple of times to establish new leader in a single election.
+        # This is racy and flaky if we just run it once. Running it multiple times
```

Comment:
I would maybe tone down this comment a bit: on an uncontended system, the test would always work: our raft implementation will deterministically win in one election without large (multi-ms) sleeps from some external source.  The retry is needed because the test _environment_ is noisy.

Reply:
Linux is not a realtime OS